samples/powerpoint/99-preview-apis/get-set-textrange.yaml

```
@@ -0,0 +1,173 @@
+order: 3
+id: powerpoint-get-set-textrange
+name: Work with text range selections
+description: 'Get, set, load, and save text range selections.'
+host: POWERPOINT
+api_set: {}
+script:
+    content: |
+        $("#getSelectedTextRange").click(() => tryCatch(getSelectedTextRange));
+        $("#setSelectedTextRange").click(() => tryCatch(setSelectedTextRange));
+        $("#changeColor").click(() => tryCatch(changeColor));
+        $("#saveTextSelection").click(() => tryCatch(saveTextSelection));
+        $("#loadTextSelection").click(() => tryCatch(loadTextSelection));
+
+        async function getSelectedTextRange() {
+          await PowerPoint.run(async (context) => {
+            const textRange = context.presentation.getSelectedTextRange();
+            textRange.load("text");
+            textRange.load("start");
+            textRange.load("length");
+            await context.sync();
+            let txtHtml = textRange.text;
+            txtHtml = txtHtml.replace(/\n/g, "<br>");
+            txtHtml = txtHtml.replace(/\r/g, "<br>");
+            txtHtml = txtHtml.replace(/\v/g, "<br>");
+            let txtExplained = textRange.text;
+            txtExplained = txtExplained.replace(/\n/g, "<font color=red>NL</font>");
+            txtExplained = txtExplained.replace(/\r/g, "<font color=red>CR</font>");
+            txtExplained = txtExplained.replace(/\v/g, "<font color=red>VV</font>");
+            let finalTable = "";
+            finalTable +=
+              "<br><table border=1 cellpadding=3 cellspacing=0><tr><td bgcolor=#3333EE><font color=white>Index</font></td><td bgcolor=#3333EE><font color=white>Id</font></td></tr>";
+            finalTable += "<tr><td>Raw</td><td>" + textRange.text + "</td></tr>";
+            finalTable += "<tr><td>Html</td><td>" + txtHtml + "</td></tr>";
+            finalTable += "<tr><td>Exp</td><td>" + txtExplained + "</td></tr>";
+            finalTable += "<tr><td>Start</td><td>" + textRange.start + "</td></tr>";
+            finalTable += "<tr><td>Length</td><td>" + textRange.length + "</td></tr>";
+            finalTable += "</table>";
+            $("#outputSpan").empty();
+            $("#outputSpan").append(finalTable);
+          });
+        }
+        async function setSelectedTextRange() {
+          await PowerPoint.run(async (context) => {
+            const shapes = context.presentation.getSelectedShapes();
+            const shapeCount = shapes.getCount();
+            await context.sync();
+            if (shapeCount.value !== 1) {
+              console.log("You must select at least one shape with text in it.");
```

consider saying `console.warning` or `console.error`
```suggestion
              console.warning("You must select at least one shape with text in it.");
```

---

This should be fixed now.